source/game/system/RaceManager.hh

```diff
@@ -47,7 +50,10 @@ public:
         f32 m_lapCompletion;
         s8 m_jugemId;
         s16 m_currentLap;
+        s8 m_maxLap;
         s8 m_maxKcp;
+        std::array<Timer, 3> m_lapTimers;
```

Comment:
This will break for custom tracks having more than 3 laps. Should we account for this now, or deal with it later?

Reply:
Ditto.